.github/workflows/qiita-ci.yml

```diff
@@ -169,17 +169,20 @@ jobs:
           conda deactivate
 
           echo "8. Setting up SSH"
-          ssh-keygen -t rsa -b 4096 -N '' -f $PWD/qiita_ware/test/test_data/test_key
+          ssh-keygen -t ed25519 -a 200 -N '' -f $PWD/qiita_ware/test/test_data/test_key
```

Comment:
Curious: why was this switch necessary?

Reply:
Reading online they say that ed25519 vs rsa (just different encoding and in theory more secure) is necessary for github actions, which it wasn't in the past; thus decided to change it as part of the many trials to make it work. My thought was to leave it as this as it supposed to be the newer way